test/fmi4c_test_fmi2.c

```diff
@@ -152,7 +154,7 @@ int testFMI2ME(fmiHandle *fmu)
             exit(1);
         }
 
-        eventIndicatorsPrev = eventIndicators;
+        memcpy(eventIndicatorsPrev, eventIndicators, sizeof(eventIndicators));
```

Comment:
Need to `#include <string.h>` on Linux for memcpy
Also get a warning about `sizeof(eventIndicators)` it is size of double*  and not the number of elements in the array as I think the intention is. Maybe use `nEventIndicators*sizeof(double)` instead